components/alert/alert.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Alert • PowertrainUI</title>
+  </head>
+  <body>
+    <main class="wrapper">
+      <nav class="wrapper__nav-top">
```

Comment:
navigation should be inside the `header` tag not `main` 

Also, don't use `main` as a wrapper, you can use `div` for general-purpose styling.

Reply:
Not focusing on docs html but valuable points noted for later!